packages/react-reconciler/src/ReactFiber.js

```diff
@@ -253,13 +253,13 @@ function FiberNode(
 
   this.alternate = null;
 
-  if (enableProfilerTimer) {
-    this.actualDuration = 0;
-    this.actualStartTime = -1;
-    this.selfBaseDuration = 0;
-    this.treeBaseDuration = 0;
-  }
+  this.actualDuration = 0;
+  this.actualStartTime = -1;
+  this.selfBaseDuration = 0;
+  this.treeBaseDuration = 0;
```

Comment:
I thought this conditional was stripped from production and production+profiling builds (as part of uglification).

Removing it will cause non-profiling production builds to increase memory size, no?

Reply:
Looking at the CJS bundles in NPM, it looks to me like the condition _is_ being removed from all but the dev builds.

Reply:
FWIW, my above comment only applies to builds where this flag is static ^

Reply:
I was testing this on the FB bundle, sorry, I should have stated. We need to somehow make this static for FB too otherwise it seems most, if not all of our fibers on FB code end up getting deoptimized on V8 :(

Reply:
It is static for production bundles– just not the dev bundle because we don't dead code eliminate it.

We choose which bundle to serve based on a gatekeeper. (We switch at the level of the react-dom bundle.)

Reply:
I remove the change, as it's not the fix. Something else must be touching those properties. Thanks for the help, I'll keep digging.